src/app/admin/dashboard/tabele/page.tsx

```
@@ -0,0 +1,653 @@
+                    * Sistem traži igrače po nazivu tima i ažurira njihove
+                    ukupne poene
+                  </p>
+                </div>
+
+                {/* JSON Input */}
+                <div className="mb-6">
+                  <label className="block text-sm font-medium text-gray-700 mb-2">
+                    JSON Podaci:
+                  </label>
+                  <textarea
+                    value={bulkUpdateData}
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)